### PR TITLE
RHEL.8 is no longer test pktgen

### PR DESCRIPTION
**Pull request by yanglei-rh**
1.Modify "server_session" to let it raise Exception

Signed-off-by: Lei Yang <leiyang@redhat.com>

ID:1829661 1710793

### CODE REVIEW

**Comment by yanglei-rh** [comment]
@PaulYuuu Please help me review it,thanks.

**Comment by PaulYuuu** [comment]
Please deal with 1710793 at the same time, they are the same issue.

**Comment by yanglei-rh** [comment]
@PaulYuuu I have deal with bug 1710793, please help me review it again.

File: qemu/tests/cfg/pktgen.cfg [hunk 1]
```diff
@@ -2,6 +2,7 @@
     no JeOS
     no Windows
     no Host_RHEL.m5, Host_RHEL.m6
```
Comment (PaulYuuu) [inline qemu/tests/cfg/pktgen.cfg:4]: `host_guest` this variant will test host <-> guest, so host should not a RHEL.8 host

Comment (yanglei-rh) [reply qemu/tests/cfg/pktgen.cfg]: I have update it, please help me review it again. Thanks.

**Comment by PaulYuuu** [comment]
LGTM

**Pull request merged**